src/pages/nft/class/_classId/index.vue

```diff
@@ -204,12 +204,15 @@ export default {
     };
   },
   computed: {
+    ...mapGetters(['LIKEPriceInUSD']),
     classId() {
       return this.$route.params.classId;
     },
     NFTPriceUSD() {
-      const price = this.currentPrice * this.purchaseInfo.price;
-      return `(${price.toFixed(3)} USD)`;
+      return this.LIKEPriceInUSD * this.purchaseInfo.price;
+    },
+    formattedNFTPriceUSD() {
+      return `(${this.NFTPriceUSD.toFixed(3)} USD)`;
```

Comment:
I think it is better to move `NFTPriceUSD` and `formattedNFTPriceUSD`  to nft mixin

Reply:
Should we rename the original `nftPriceInLIKE` and `nftPriceInUSD` in NFT mixin which are fetched from strip API?

Reply:
My suggestion,
- `nftPriceInLIKE` can be removed
- keep `nftPriceInUSD` unchanged for now
- Add `NFTPriceUSD`
- Add `formattedNFTPriceUSD `

I think it is better to have a large scale renaming in another PR